bodhi/server/services/graphql.py

```diff
@@ -17,18 +17,56 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 """Defines API endpoints related to GraphQL objects."""
 from cornice import Service
+from webob_graphql import serve_graphql_request
+import graphene
```

Comment:
Let's move this import at the top of the import 
```suggestion
import graphene
```
```suggestion
import graphene

from cornice import Service
from webob_graphql import serve_graphql_request
```